atomix/cluster/src/main/java/io/atomix/raft/storage/log/entry/InitializeEntry.java

```diff
@@ -25,9 +25,4 @@
  * following leader changes. Initialize entries are always the first entry to be committed at the
  * start of a leader's term.
  */
-public class InitializeEntry extends TimestampedEntry {
-
-  public InitializeEntry(final long term, final long timestamp) {
-    super(term, timestamp);
-  }
-}
+public class InitializeEntry implements RaftEntry {}
```

Comment:
Is there any reason why we're removing the timestamp for this but keeping it in the ConfigurationEntry?

Reply:
Can we rename this to `InitialEntry`? We alternate between the two names throughout the code so we can take this opportunity to make it consistent

Reply:
> Is there any reason why we're removing the timestamp for this but keeping it in the ConfigurationEntry?

Timestamp in ConfigurationEntry is used elsewhere. Timestamp for the other entries are not required.